src/pymedphys/logfile/_level1/logfileindex.py

```diff
@@ -173,7 +173,7 @@ def file_ready_to_be_indexed(cursors, filehash_list, to_be_indexed_dict,
         try:
             delivery_details = get_mosaiq_delivery_details(
                 cursors[server], header.machine, mosaiq_string_time,
-                header.field_label, header.field_name)
+                header.field_label, header.field_name, buffer=120)
```

Comment:
This here is set so that by default a 2 minute buffer is placed around the logfile identification within Mosaiq. This helps account for the clock on the Linac being misaligned.